go.mod

```diff
@@ -5,14 +5,13 @@ go 1.20
 require (
 	github.com/container-storage-interface/spec v1.8.0
 	github.com/containers/image/v5 v5.27.0
-	github.com/docker/cli v23.0.5+incompatible
+	github.com/docker/cli v24.0.0+incompatible
 	github.com/evanphx/json-patch v5.6.0+incompatible
 	github.com/go-logr/logr v1.2.4
-	github.com/google/go-containerregistry v0.15.2
+	github.com/google/go-containerregistry v0.16.1
 	github.com/klauspost/compress v1.16.7
 	github.com/mattn/go-sqlite3 v1.14.17
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.1.0-rc3
```

Comment:
this change related to my prev PR 
https://github.com/Dynatrace/dynatrace-operator/commit/a803e7787c7d569cf5d644af71d4d1b1050d6b42#diff-5dc4e3fe14beba46855f844e201c73d30a69fe0884f48911a3ad94cbb16b61feL12

so we don't need it anymore.